rns_shell/compositor/layers/Layer.cpp

```diff
@@ -8,7 +8,6 @@
 
 #include "compositor/layers/Layer.h"
 #include "compositor/layers/PictureLayer.h"
-
 namespace RnsShell {
```

Comment:
if no change in this file , should probably revert this file to not have it in the PR